bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -289,7 +290,13 @@ <h2>{{ _('You deserve peace of mind everywhere.') }}</h2>
         </div>
 
         <div class="show-fxa-supported-signed-out">
-          {{ fxa_email_form(entrypoint='accounts-page', utm_source='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': 'accounts-page', 'medium': 'referral'}, button_class='mzp-c-button mzp-t-primary mzp-t-product', button_text=_('Create an Account')) }}
+          {{ fxa_email_form(
+            entrypoint='mozilla.org-fxa_benefits',
+            utm_source='mozilla.org-fxa_benefits',
+            utm_params={'referrer': 'https://www.mozilla.org/en-US/firefox/accounts/'},
```

Comment:
Is there a need to add `referrer` here? From our convo yesterday, it sounded like this is only necessary if `entrypoint` is missing. If we do require adding this, we should make sure it's adding the correct locale in the URL.

Reply:
really good point/catch. Do we have logic to add `referrer` if `entrypoint` is missing? If not I think that would make a good issue.

Reply:
I don't know of anywhere where we don't pass `entrypoint` already, so there's no logic for referrer afaik.